aea/protocols/generator.py

```diff
@@ -826,6 +852,7 @@ def _message_class_str(self) -> str:
             cls_str += self._import_from_custom_types_module()
         else:
             cls_str += "\n{}\n".format(self._import_from_custom_types_module())
+        cls_str += "\nlogger = logging.getLogger(__name__)"
```

Comment:
@marcofavorito this is an issue I believe as the logger will have to be given the correct name. Can you advise?

Reply:
Yes, maybe set the name as:
```
packages.protocols.<protocol_name>.serializatino
```

Reply:
So the generator should produce this line?
logger = logging.getLogger(packages.protocols.<protocol_name>.serialization)

Note, this logger is for the message class though.

Reply:
@marcofavorito it should be the following?:
```
logger = logging.getLogger(packages.<author>.protocols.<protocol_name>.message)
```

Reply:
so that means the protocols generated by the tool are vendorized by convention?

Reply:
No, my bad, we now have `author` in every import path, regardless of vendorized or not.

Reply:
Done